encoding/video-artifacts.md

```diff
@@ -261,6 +261,17 @@ it's better to do this under 16bit to avoid precision problems.
 
 ![](images/overflow_notice.jpg)
 
+Here's some examples if you want to use `std.Expr()`.
+
+```py
+# We suppose "src16" is a 16bit video clip and we only process Y plane in this example as well as bypass the UV planes.
+# I have to underline that this example is processed under 16bit and I don't recommend to do this at any lower precisions.
```

Comment:
Please try not to use "I" in either text or comments. Passive forms are probably the way to go here. I can also do this, if you'd like me to.

Reply:
Great! Please go ahead and do this.